config/locales/en.yml

```diff
@@ -3,3 +3,5 @@ en:
     required: "%{param} is required"
     invalid_type: "%{param} must be a valid %{type}"
     invalid_in: "%{param} is an invalid value"
+    more_than_max: "%{param} cannot be more than maximum"
```

Comment:
need to display the maximum/minumum here to users, so they know what the limits are,  as %{value}

Reply:
I've used the same implementation as `validate_inclusion`
```
            message: I18n.t("validate_params.less_than_min", param: error_param_name),
            min: @options[:min]
```
A separate key:value pair is used for that